main/sceneManager.cpp

```diff
@@ -1,26 +1,66 @@
 #include "main/sceneManager.h"
 
-using Tri = objects::TriangleObj;
+SceneManager::SceneManager()
+{
+    createCollectors();
+}
+
+std::vector<objects::EnergyCollector *> SceneManager::getEnergyCollectors()
+{
+    std::vector<objects::EnergyCollector *> temp;
+    temp.reserve(_energyCollectors.size());
+    for (const auto &t : _energyCollectors)
+    {
+        temp.push_back(t.get());
+    }
+    return temp;
+}
+
+std::vector<objects::TriangleObj *> SceneManager::getModelTriangles()
+{
+    std::vector<objects::TriangleObj *> temp;
+    temp.reserve(_modelTriangles.size());
+    for (const auto &t : _modelTriangles)
+    {
+        temp.push_back(t.get());
+    }
+    return temp;
+}
 
-void IamSimulator()
+// PRIVATE METHODS
+
+void SceneManager::createCollectors()
```

Comment:
This function is lacking a comment what are you actually trying to do here, i.e what kind of arrangement you are trying to get?

Reply:
should I write this in .h file like this? 
![image](https://user-images.githubusercontent.com/45213495/96496388-79c81380-1249-11eb-8a2f-b8892fec2e1d.png)

Reply:
https://google.github.io/styleguide/cppguide.html#Function_Comments

Have you actually read the styleguide? ;-)

Reply:
I havent done yet, because as I said I will read it after I will finish the book... and I still didnt finished it :P

Reply:
done